packages/components/mcg/ObjectBucketClaim.tsx

```diff
@@ -262,9 +265,13 @@ export const OBCListPage: React.FC<PageProps> = (props) => {
   return (
     <>
       <Modal {...modalProps} />
-      <ListPageHeader title={t('ObjectBucketClaims')}>
+      <ListPageHeader
+        title={t('{{obcDisplayText}}', {
+          obcDisplayText: OBJECT_BUCKET_CLAIMS,
+        })}
+      >
         <ListPageCreateLink to={createLink}>
-          {t('Create ObjectBucketClaim')}
+          {t('Create new')}
```

Comment:
```suggestion
          {t('Create ObjectBucketClaim')}
          OR,
          {t('Create new ObjectBucketClaim')}
```
based upon how we show at other places.